src/Analyzer/FileVisitor.php

```diff
@@ -154,6 +161,12 @@ public function leaveNode(Node $node): void
 
             $this->classDescriptions[] = $classDescription;
         }
+
+        if ($node instanceof Node\Stmt\Enum_ && null !== $this->classDescriptionBuilder) {
```

Comment:
Here we treat enum as a class which is fine for now. I am wondering if we should add a method like isEnum or similar

Reply:
What is the benefit to add the method `isEnum` now?
I think it could be fine.